ghost/admin/app/components/gh-context-menu.js

```diff
@@ -36,12 +88,14 @@ export default class GhContextMenu extends Component {
 
     @action
     open() {
-        this.isOpen = true;
+        this.setState('open');
     }
 
     @action
     close() {
-        this.isOpen = false;
+        if (this.state === 'open') {
```

Comment:
We add this check because you can only "close" the menu when it is open. However you can transition to the `default` state from any other state.